spec/index.html

```diff
@@ -10369,7 +10369,151 @@ <h3>Evaluation Semantics</h3>
 eval(D(G), Slice(L, start, length)) = Slice(eval(D(G), L), start, length)
             </pre>
           </div>
+
+          <!-- ValuesInsertion -->
+          <section>
+            <h3>Values Insertion and `EXISTS`</h3>
+            <p>The following sections contain draft of material for a revised
+              "`exists`" operation.
+            </p>
+            <p>
+              Background: <a href="https://github.com/w3c/sparql-dev/blob/main/SEP/SEP-0007/sep-0007.md">SPARQL CG SEP-0007</a>
+            </p>
+
+            <section>
+              <h4>Syntax Restriction</h4>
+              <p><i>Additional [[[#sparqlGrammar]]] note:</i></p>
+                <blockquote>
+                Any variable that is assigned to in the graph pattern of `EXISTS`/`NOT EXISTS` must not be in-scope.
+                This applies to `BIND`, variables introduced by `AS` in a `SELECT` clause, variables in a `VALUES`
+                clause, and variables intrucded by `AS` in `GROUP BY`.
```

Comment:
If this is a copy-paste blockquote, then the original text should also be edited and/or flagged as an erratum with these corrections.
```suggestion
                Any variable that is assigned in the graph pattern of `EXISTS`/`NOT EXISTS` must not be in-scope.
                This applies to `BIND`, variables introduced by `AS` in a `SELECT` clause, variables in a `VALUES`
                clause, and variables introduced by `AS` in `GROUP BY`.
```